tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -70,10 +70,10 @@ Standard output:
 @fluentui/docs: [XX:XX:XX] Requiring external module @uifabric/build/babel/register
 @fluentui/docs: [XX:XX:XX] Working directory changed to /office-ui-fabric-react
 @fluentui/docs: [XX:XX:XX] Using gulpfile /office-ui-fabric-react/gulpfile.ts
-@fluentui/docs: DocToccing single file "/office-ui-fabric-react/.github/CONTRIBUTING.md" for github.com.
```

Comment:
@weswigham is this the output whose order isn't stable? If not, I'll merge this PR because the other two changes look stable(ish).